src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -39,28 +55,201 @@ export default function NurseryWithdrawals(props: NurseryWithdrawalsProps): JSX.
   const theme = useTheme();
   const contentRef = useRef(null);
   const classes = useStyles();
-  const [nurseryWithdrawals, setNurseryWithdrawals] = useState<SearchResponseElement[]>();
+  const [searchResults, setSearchResults] = useState<SearchResponseElement[] | null>();
   const history = useHistory();
+  const [searchValue, setSearchValue] = useState('');
+  const debouncedSearchTerm = useDebounce(searchValue, 250);
+  const [filters, setFilters] = useForm<NurseryWithdrawalsFiltersType>({});
+  const [species, setSpecies] = useState<Species[]>();
+  const [snackbar] = useState(useSnackbar());
+
+  const onWithdrawalClicked = (withdrawal: any) => {
+    history.push({
+      pathname: APP_PATHS.NURSERY_REASSIGNMENT.replace(':deliveryId', withdrawal.delivery_id),
+    });
+  };
 
   useEffect(() => {
-    const getNurseryWithdrawals = async () => {
-      if (organization) {
-        const withdrawals = await listNurseryWithdrawals(organization.id, []);
-        if (withdrawals) {
-          setNurseryWithdrawals(withdrawals);
-        }
+    const populateSpecies = async () => {
+      const result = await getAllSpecies(organization.id);
+      if (result.requestSucceeded) {
+        setSpecies(result.species);
+      } else {
+        snackbar.toastError();
```

Comment:
Thanks! Can we iterate later on what a good error message might be? I do agree with you that it could be confusing if we don't give context.
Something like `couldn't load species list for search`